consensus/istanbul/core/backlog.go

```diff
@@ -54,7 +57,19 @@ func (c *core) checkMessage(msgCode uint64, view *istanbul.View) error {
 		return errFutureMessage
 	}
 
+	// as soon as we move on to the next sequence, we discard any messages from
+	// a past sequence as invalid old messages unless they are Commits of the
+	// previous sequence
 	if view.Cmp(c.currentView()) < 0 {
+		// let commits from previous sequence through so that the handler adds them
+		// to the view's parent commits (the round does not matter in this case,
```

Comment:
nit: I would change this comment slightly to:

`Let commits from the previous sequence through so that the handler adds them to the view's parent commits. The round does not matter in this case, as we only care that the message's committed seal matches the previous block.`